lib/src/actions/actions.dart

```diff
@@ -2752,6 +2941,9 @@ abstract class ScalePurificationIDTFieldsAssign
 
   static Serializer<ScalePurificationIDTFieldsAssign> get serializer =>
       _$scalePurificationIDTFieldsAssignSerializer;
+
+  @override
+  String short_description() => "assign scale purification idf fields";
```

Comment:
should this say "IDT" instead of "idf"?

Reply:
https://github.com/UC-Davis-molecular-computing/scadnano/pull/752/commits/0e2fbfca5847104ca37b9269a2466f88658d0ab0

Reply:
https://github.com/UC-Davis-molecular-computing/scadnano/pull/752/commits/79c56f92f398633d1354e304c264fce61aa25a01